features/support/env.rb

```diff
@@ -4,9 +4,6 @@
 Before do
   puts "Killing server..."
   system "for i in `ps axu | grep 'server.rb' | grep ruby | awk '{print $2}'`; do kill -9 $i; done > /dev/null 2>&1"
-  puts "Killing handler..."
```

Comment:
did git bisect the broken test suite (`bundle exec cucumber`) down to 8dc3e26 and confirmed that this step is causing the error described in the commit message.

Removing it is not a fix, as it supposes to remove the TCP server Cucumber feature. Most of the feature is quite low-level however, and I think that the benefits of a working test suite are outweigh the cost.

I'll try to find a better was the default host and port of the Dredd hooks server as it is part of the API that Dredd expects to be available.